src/main/java/io/github/syst3ms/skriptparser/expressions/ExprAnswer.java

```diff
@@ -0,0 +1,60 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.effects.EffAsk;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Get back the last entered answer in a {@link EffAsk} effect.
+ * This will return the last line of the console.
+ *
+ * @name Last Answer
+ * @type EXPRESSION
+ * @pattern [the] [last] (answer|response)
+ * @since ALPHA
+ * @author ItsTheSky
+ */
+public class ExprAnswer implements Expression<String> {
+
+    static {
+        Parser.getMainRegistration().addExpression(
+                ExprAnswer.class,
+                String.class,
+                true,
+                "[the] [last] (answer|response)"
```

Comment:
Not sure if we should add 'input' as an option here. It might break with [ExprSingleParameter](https://github.com/SkriptLang/skript-parser/blob/master/src/main/java/io/github/syst3ms/skriptparser/expressions/ExprSingleParameter.java).

Reply:
i think it's would be better to not create collisions right now and wait that parser can handle both syntaxs